docs/fastdds/dds_layer/topic/typeSupport/includes/fastddsgen.rst

```diff
@@ -0,0 +1,59 @@
+.. _dds_layer_topic_fastddsgen:
+
+Fast DDS-Gen for data types source code generation
```

Comment:
I think the section is to hidden. I'd actually put it as a children of Topic section